docs/management/connectors/action-types/cases-webhook.asciidoc

```diff
@@ -0,0 +1,384 @@
+
+_Note: We are using Jira as an example third party. However, if you wish to connect with Jira, using the <<jira-action-type, preconfigured Jira connector>> will simplify your setup._
+
+[float]
+====  Step 1 - Set up connector
+
+In the Webhook - Case Management connector create flyout, we begin by entering a *Name* for the connector - `Jira Test Connector`. We are going to use basic authentication in this example, so we keep the *Require authentication* checkbox checked and enter the *Username* and *Password* for the test instance, `test-user@elastic.co` and `notarealpassword`. We will not be setting any *Headers* for the requests.
+
+[role="screenshot"]
+image::management/connectors/images/cases-webhook-step1.png[Webhook - Case Management connector Step 1, Jira example]
+
+[float]
+====  Step 2 - Create case
+
+Taking a look at the Jira https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issues/#api-rest-api-2-issue-post[Create issue method documentation], we can find what values we will need to fill step 2.
+
+Jira create issue request method: `POST`
+
+Jira create issue request url: `/rest/api/2/issue`
+
+Jira create issue request body:
+[source,json]
+--
+{
+    "fields": {
+        "summary": "Main order flow broken",
+        "description": "Order entry fails when selecting supplier.",
+        "labels": ["bugfix",  "blitz_test"],
+        "project":{"key":"PROJ-123"},
+        "issuetype":{"id":"10000"}
+    }
+}
+--
+
+Jira create issue response body:
+[source,json]
+--
+{
+  "id": "10000",
+  "key": "ED-24",
+  "self": "https://your-domain.atlassian.net/rest/api/2/issue/10000",
+  "transition": {
+    "status": 200,
+    "errorCollection": {
+      "errorMessages": [],
+      "errors": {}
+    }
+  }
+}
+--
+In the screen capture below, on Step 2 we enter `POST` as the *Create Case Method* and `https://testing-jira.atlassian.net/rest/api/2/issue` as the **Create Case Url**. In our example Jira instance, the project key is "ROC" and the issuetype id is "10024". We have entered the Jira request JSON as the *Create Case Object*, updating the project key to "ROC" and the issuetype id to "10024". We then use the Case variable selector to enter where we will map the Kibana case title, Kibana case description, and Kibana case tags. The Jira response body contains an id with the JSON key of "id", so we enter `id` as the *Create Case Response - Case ID Key*.
+[role="screenshot"]
+image::management/connectors/images/cases-webhook-step2.gif[Webhook - Case Management connector Step 2, Jira example]
+
+[float]
+====  Step 3 - Get case information
+
+Next we'll need to look at Jira's https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issues/#api-rest-api-2-issue-issueidorkey-[Get issue method documentation] to find the values for step 3. In the GET response JSON below, we thinned out some null/unrelated data so that we can focus on the fields we need.
```

Comment:
```suggestion
Next we'll need to look at {jira}'s https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issues/#api-rest-api-2-issue-issueidorkey-[Get issue method documentation] to find the values for this step. In the GET response JSON below, we thinned out some null and unrelated data so that we can focus on the fields we need.
```